Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -697,7 +688,7 @@ protected virtual void Dispose(bool disposing)
 
             if (disposing)
             {
-                Stop();
+                // TODO:
```

Comment:
eh?

Reply:
Nothing to dispose atm

Reply:
Then why add a comment at all? Just say `// nothing to dispose here`

Reply:
In the ideal world this should dispose something, or not be disposable

Reply:
In this case, let's fill it with `// TODO: decide if it needs to be Disposable`?.. I mean, an empty `TODO` comment is kinda useless...

Reply:
cc @Bond-009